test/extended/cli/mustgather.go

```diff
@@ -126,6 +124,11 @@ var _ = g.Describe("[sig-cli] oc adm must-gather", func() {
 	})
 
 	g.It("runs successfully for audit logs", func() {
+		// On IBM ROKS, events will not be part of the output, since audit logs do not include control plane logs.
```

Comment:
Why is this new here?

Reply:
> Why is this new here?

it was down below, but it nerfed the test, so this just nerfs it earlier.